### DISCUSSION TITLE
Support installation for one drive that's installed in the home dir

### DISCUSSION THREAD

**Discussion by DarthMDev** [bpozdena/OneDriveGUI#81] [Ideas]
For example, with package managers like Homebrew that allows it to stay persistent through steam deck updates.
Would help a lot for steam deck users.
Currently, when having it outside /usr/local/bin, it doesn't detect it as installed.

**Comment by bpozdena** [comment]
This seems doable.  It could also be helpful for testing with different onedrive client versions. 

I don't know what limitations SteamDeck has and have some questions:
1)Are you running the GUI from AppImage or source? 
2)In what path do you normally store the onedrive client executable?
2)Does SteamDeck allow you to create persistent a symlink in  `/usr/local/bin`?

**Reply by DarthMDev** [reply]
1. the appimage i have it in ~/Applications
2. Its in /home/linuxbrew/.linuxbrew/bin/onedrive other ppl might install it in somewhere like ~/.local/bin for ex 
3. I dont think so it , it would override it with the updated image on the next update , same as installing it

**Reply by bpozdena** [reply]
Thanks for the info. Please check #82 for next update. I will provide a test AppImage there in a few minutes.
